test/Microsoft.Data.Analysis.Tests/DataFrame.IOTests.cs

```diff
@@ -140,12 +144,59 @@ void ReducedRowsTest(DataFrame reducedRows)
         }
 
         [Fact]
-        public void TestReadCsvNoHeader()
+        public void TestReadCsvSplitAcrossMultipleLines()
         {
-            string data = @"CMT,1,1,1271,3.8,CRD,17.5
-CMT,1,1,474,1.5,CRD,8
-CMT,1,1,637,1.4,CRD,8.5
-CMT,1,1,181,0.6,CSH,4.5";
+            string CMT = @"""C
+MT""";
+            string verifyCMT = @"C
+MT";
+            string data = @$"{CMT},1,1,1271,3.8,CRD,17.5
+{CMT},1,1,474,1.5,CRD,8
+{CMT},1,1,637,1.4,CRD,8.5
+{CMT},1,1,181,0.6,CSH,4.5";
+
+            Stream GetStream(string streamData)
```

Comment:
Instead of duplicating this in all the tests, why not just make it a regular `private static` method?